docs/book/v1/intro.md

```diff
@@ -1,8 +1,20 @@
 # laminas-servicemanager-migration
```

Comment:
Navigation entry and title / first headline of the page must be the same:

https://github.com/laminas/laminas-servicemanager-migration/blob/751890e8ac17cb9f806359945845abe6fa02b200/mkdocs.yml#L5

```suggestion
# Introduction
```